packages/react-ui-core/src/List/__tests__/List-test.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import theme from './mocks/theme'
+import ThemedList from '../List'
+
+const List = ThemedList.WrappedComponent
+
+describe('List', () => {
+  let wrapper
+
+  beforeEach(() => {
+    wrapper = mount(
+      <List />,
+    )
+  })
+
+  it('renders a List', () => {
+    wrapper.setProps({ theme })
+    expect(wrapper.find(List).length).toBe(1)
+  })
+
+  it('applies a theme classname', () => {
+    wrapper.setProps({ theme })
+    const className = wrapper.find('ul').prop('className')
+    expect(className).toContain('List')
+  })
+
+  it('passes through props to List Item', () => {
+    wrapper.setProps({ listItemNodeType: 'span', theme })
+    expect(wrapper.find('List').prop('listItemNodeType')).toEqual('span')
+  })
+
+  it('accepts custom node type', () => {
+    wrapper.setProps({ nodeType: 'div', theme })
+    expect(wrapper.find('div').length).toBe(1)
+  })
+
+  it('accepts custom orientation', () => {
+    wrapper.setProps({ orientation: 'horizontal', theme })
+    const className = wrapper.find('ul').prop('className')
+    expect(className).toContain('horizontal')
```

Comment:
You probably want `toContain('List-horizontal')` just so we know that exact class exists.